testng-eclipse-plugin/src/main/org/testng/eclipse/launch/TestNGLaunchConfigurationDelegate.java

```diff
@@ -355,18 +355,6 @@ protected VMRunnerConfiguration createVMRunner(
   }
 
   @Override
-  public String getVMArguments(ILaunchConfiguration configuration)
-      throws CoreException {
-    StringBuilder vmArgs = new StringBuilder();
-    vmArgs.append(super.getVMArguments(configuration));
-
-    vmArgs.append(" ");
-    vmArgs.append(ConfigurationHelper.getJvmArgs(configuration));

```

Comment:
I just deployed a new plug-in with this fix.

-- 
Cédric


On Wed, Dec 23, 2015 at 6:34 PM, Nick Tan <notifications@github.com> wrote:

> In
> testng-eclipse-plugin/src/main/org/testng/eclipse/launch/TestNGLaunchConfigurationDelegate.java
> <https://github.com/cbeust/testng-eclipse/pull/212#discussion_r48348948>:
>
> > @@ -355,18 +355,6 @@ protected VMRunnerConfiguration createVMRunner(
> >    }
> >
> >    @Override
> > -  public String getVMArguments(ILaunchConfiguration configuration)
> > -      throws CoreException {
> > -    StringBuilder vmArgs = new StringBuilder();
> > -    vmArgs.append(super.getVMArguments(configuration));
> > -
> > -    vmArgs.append(" ");
> > -    vmArgs.append(ConfigurationHelper.getJvmArgs(configuration));
>
> the 2nd call retrieves org.eclipse.jdt.launching.VM_ARGUMENTS and append
> to testng JVM process
> so, the fix is simply call ConfigurationHelper.getJvmArgs(configuration)
> to make sure only have one org.eclipse.jdt.launching.VM_ARGUMENTS append
> to testng
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cbeust/testng-eclipse/pull/212/files#r48348948>.
>